common/lib/dependabot/file_fetchers/base.rb

```diff
@@ -518,6 +518,16 @@ def _clone_repo_contents(target_directory:)
               git clone --no-tags --no-recurse-submodules --depth 1#{br_opt} #{source.url} #{path}
             CMD
           )
+          if source.commit
+            # This code will only be called for testing. Production will never pass a commit
+            # since Dependabot always wants to test the latest commit on a branch.
```

Comment:
typo?
```suggestion
            # since Dependabot always wants to use the latest commit on a branch.
```

Reply:
Yep that would make more sense. Brain getting ahead of typing again!

Reply:
I'll put up the PR since you're headed out for the weekend, that way we don't lose track of it and confuse future devs...

Reply:
https://github.com/dependabot/dependabot-core/pull/5847